wai-aria/role/role_none_conflict_resolution.tentative.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Role None Conflict Resolution Verification Tests</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/resources/testdriver.js"></script>
+    <script src="/resources/testdriver-vendor.js"></script>
+    <script src="/resources/testdriver-actions.js"></script>
+    <script src="/wai-aria/scripts/aria-utils.js"></script>
+  </head>
+  <body>
+    
+    <!-- no conflict, since not focusable; inner button should be exposed as generic - children presentational - //https://github.com/web-platform-tests/interop-accessibility/issues/161 -->
+    <h2>ARIA role="button" nesting ARIA role="button" (no presentational roles conflict resolution, since not focusable)</h2>
+    <div role="button">
+      <span>outer button</span>
+      <span role="button" class="ex-generic" data-testname="non-focusable ARIA[role button] nesting non-focusable ARIA[role button]">Inner button</span>
+    </div>
+
+    <!-- conflict arise, since inner button is focusable - //https://github.com/web-platform-tests/interop-accessibility/issues/161-->
+    <h2>HTML button nesting HTML button</h2>
+    <div id="button-container">
+      <button id="outer-button">
+        <span>outer button</span>
+        <span id="inner-button-container"></span>
+      </button>
+    </div>
+
+    <script>
+      AriaUtils.verifyRolesBySelector(".ex");
+      AriaUtils.verifyGenericRolesBySelector(".ex-generic");
+
+      //https://github.com/web-platform-tests/interop-accessibility/issues/161 starts
+      const innerButton = document.createElement("button");
+      innerButton.textContent = " (inner button)";
+      innerButton.id = "inner-button";
+      document.getElementById("inner-button-container").appendChild(innerButton);
```

Comment:
Do you need to change a change of the same element, or would testing the before/after states of separate elements be sufficient?

Reply:
I’m not entirely sure I understand the question. However, if I’m interpreting it correctly, the issue arises because browsers automatically adjust nested buttons by moving the inner button outside the parent button. To address this, I’m using JavaScript to dynamically inject a button inside another button.

Reply:
IIRC, you've moved the `ARIAUtils.verify…()` calls after this DOM modification, which is another was to address the concern I alluded to insufficiently... No further modification is necessary. Thanks. Okay to mark this thread as resolved.